src/response/result.ts

```diff
@@ -0,0 +1,23 @@
+import { GuardName } from "../guard/constants.js";
+
+export enum ResponseMatch {
+    YES = "YES",
+    NO = "NO"
+}
```

Comment:
I already did not understand why we had that in Python. What's wrong with using a boolean value here?

Reply:
I had it earlier with boolean but, this is better, as its better for reading and writing code logic in the guard processor and its very specific for us.

Reply:
yeah, I don't see that. An enum now turns this at every corner into a 3x branch check vs a 2x branch check with a boolean. Anyways, I don't care too much on this one right now.